pkg/apis/cnsoperator/cnsvolumemetadata/v1alpha1/doc.go

```diff
@@ -0,0 +1,5 @@
+// +k8s:deepcopy-gen=package
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=cns.vmware.com
+
+package v1alpha1
```

Comment:
What generally goes into these files? Is it the documentation of the API?

Reply:
Yes, however the code generator tags above are the main reason I added these files.

I could add those tags anywhere else in the package, but I chose to add these files in order to follow the existing pattern (`pkg/apis/migration/v1alpha1/doc.go` and `pkg/apis/storagepool/cns/v1alpha1/doc.go`).

Reply:
Makes sense. lets keep as is.